src/onefetch/cli.rs

```diff
@@ -115,6 +116,16 @@ impl Cli {
                 .possible_values(&possible_backends)
                 .help("Which image BACKEND to use."),
         )
+        .arg(
+            Arg::with_name("image-colors")
+                .long("image-colors")
+                .value_name("NUM")
+                .takes_value(true)
+                .max_values(1)
+                .possible_values(&["16", "32", "64", "128", "256"])
```

Comment:
@CephalonRho Can you confirm that these are the possible values that should be used?

Reply:
Any number above 0 should work. These seem to be chosen arbitrarily, but they are values that should provide a reasonable looking output.
A solution using a [custom validator](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.validator) would be perfect, but it's probably not worth the effort.